frontend/src/utils/emails/template-parser.ts

```diff
@@ -0,0 +1,69 @@
+// A naive template parser for the .template email files
+
+type EmailTemplate = {
+  subject: string
+  body: string
+}
+
+async function readFile(filename: string): Promise<string> {
+  return fetch(filename).then((r) => r.text())
```

Comment:
Good that `fetch` does work here! But the template files must be under `/public/` (not `/src/utils/emails/bodies/`)to be served by whatever's hosting the website